consts/api/index.ts

```diff
@@ -0,0 +1,23 @@
+import type { Method } from "axios";
+
+interface HttpMethod {
+  [key: string]: Method;
+}
+export const HTTP_METHOD: HttpMethod = {
+  GET: "get",
+  POST: "post",
+  PUT: "put",
+  PATCH: "patch",
+  DELETE: "delete",
+};
+
+export const HTTP_ERROR = {
+  BAD_REQUEST: 400,
+  UNAUTHORIZED: 401,
+  PAYMENT_REQUIRED: 402,
+  FORBIDDEN: 403,
+  NOT_FOUND: 404,
+  INTERNAL_SERVER_ERROR: 500,
+  NOT_IMPLEMENTED: 501,
+  BAD_GATEWAY: 502,
+};
```

Comment:
as const 로 불필요한 타입을 삭제하고 타입을 더 구체화 할 수 있을 거 같아
현재는 모든 value는 string일 테니까
더 확실히 해보면 좋을 거 같아

```suggestion
export const HTTP_METHOD = {
  GET: "get",
  POST: "post",
  PUT: "put",
  PATCH: "patch",
  DELETE: "delete",
} as const;

export const HTTP_ERROR = {
  BAD_REQUEST: 400,
  UNAUTHORIZED: 401,
  PAYMENT_REQUIRED: 402,
  FORBIDDEN: 403,
  NOT_FOUND: 404,
  INTERNAL_SERVER_ERROR: 500,
  NOT_IMPLEMENTED: 501,
  BAD_GATEWAY: 502,
} as const
```

Reply:
좋네 반영했어!